index.html

```diff
@@ -52,7 +55,93 @@ <h3 class="heading">Flat 40% off for neogrammers
             </div>
         </section>
     </main>
-    <script src="./scripts/nav.js"></script>
+    <section class="featured-section">
+        <h3 class="heading heading-avatar heading-card heading-ecom">Featured</h3>
+        <div class="featured-product">
+        <section>
+                <div class="card-container">
+                    <div class="image-div">
+                        <div class="overlay-image">
+                            <img class="card-image"
+                                src="https://saree.b-cdn.net/media/catalog/product/cache/b338d39772b77a6788c692c0fe37ef74/s/a/saci7054a-red-banarasi-silk-woven-saree-with-lace.jpg"
+                                alt="error" />
```

Comment:
Here you have added alt but it should add information about the image. because if your img fails to load user can see this alt on screen and having a meaningful alt will help.